src/workflows/steps.js

```diff
@@ -468,3 +470,152 @@ export function setPromote( state ) {
 export function verifyConflictResolution() {
 	return git.checkConflictMarkers();
 }
+
+export function getPackageScope( state ) {
+	if ( state.qa && typeof state.qa === "boolean" ) {
+		state.qa = "@lk";
+	} else if ( state.qa && state.qa.charAt( 0 ) !== "@" ) {
+		state.qa = `@${ state.qa }`;
+	}
+
+	return Promise.resolve();
+}
+
+export function getScopedRepos( state ) {
+	const { configPath, qa: scope } = state;
+	let content = {};
+	try {
+		content = util.readJSONFile( configPath );
+
+		let repos = Object.keys( content.devDependencies ).filter( key => key.includes( scope ) );
```

Comment:
you may want to use the lodash `get` method and default to an empty object OR check that `devDependencies` is null or undefined and default the value another way. Otherwise, it could BLOW up if it's not provided... although honestly our projects should have both `devDependencies` and `dependencies`